code/nebula/neb.cpp

```diff
@@ -191,62 +201,98 @@ void neb2_init()
 
 		// poofs
 		while (required_string_one_of(3, "#end", "+Poof:", "$Name:")) {
-
+			bool create_if_not_found = true;
 			if (Poof_info.size() < MAX_NEB2_POOFS) {
```

Comment:
This will need to be moved somewhere else I think. Even if we are at max poofs, these entries could merely be modifying them, which should still be ok. I guess where the `push_backs` are actually, happening just skip them in that case?

Reply:
Ah yeah. Good catch.

Reply:
and done